dydx-testnet-4/gentx/gentx-Provalidator.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"Provalidator","identity":"3A7D5C9B0B88BEA1","website":"https://provalidator.com","security_contact":"zenon@provalidator.com","details":"Supporting Blockchain Infrastructure"},"commission":{"rate":"1.000000000000000000","max_rate":"1.000000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx1fx94m3t4q6jtrk7e9su8wqduap04s7hf0wrndy","validator_address":"dydxvaloper1fx94m3t4q6jtrk7e9su8wqduap04s7hf2s49x4","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"DBcYxeoafX8x0iHvu+SNBNavOFZ2WpOjd8T6fBKUcXo="},"value":{"denom":"adv4tnt","amount":"1000000000000000000000000"}}],"memo":"4cee1bb78193f7dfda241d0c2653badd2a63d6e1@139.162.91.224:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AjZ6bszonSZYfDMS0KtSpWfJqJyDOqVeuGPQjwktsQms"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["49LCE/PdBKIbThAMAg2CuT6gV48Cni5AzRSyV14YwrADLVUaKQ4I7mS11H38Q8cJhXuIA7fow646ryHtnG38MA=="]}
```

Comment:
As per the [Prepare for Genesis](https://v4-teacher.vercel.app/validators/preparing_for_genesis) guide the Public Testnet 4 self-delegation amount is 500000000000000000000000adv4tnt

Reply:
I changed amount. Check again please. Thank you!